.github/workflows/ov-gpu-nightly.yml

```diff
@@ -0,0 +1,82 @@
+name: OpenVino GPU Nightly Test
+
+on:
+  workflow_dispatch:
+
+permissions: read-all
+
+jobs:
+  nightly:
+    name: OpenVino GPU Nightly
+    runs-on: [self-hosted, l0]
+
+    env:
+      GH_TOKEN: ${{ github.token }}    
+
+    steps:
+      - name: "Checkout Graph Compiler"
+        uses: actions/checkout@v4
+        with:
+          repository: 'intel/graph-compiler'
+          path: 'graph-compiler'
+
+      - name: Setup MLIR Python bindings
+        id: setup-mlir-python-bindings
+        uses: ./graph-compiler/.github/actions/setup-mlir-python-bindings
+
+      - name: Build LLVM with IMEX
+        working-directory: graph-compiler
+        run: |
+          scripts/compile.sh --dev --llvm --imex
+          echo LLVM_INST_PATH=$(pwd)/externals/llvm-project/build >>$GITHUB_ENV
+
+      - name: "Checkout OpenVino"
+        uses: actions/checkout@v4
+        with:
+          repository: 'dchigarev/openvino'
+          ref: 'gc-gpu'
+          path: 'openvino'
+          submodules: true
+          fetch-depth: 0
+          recursive: true
+      
+      - name: Build OpenVino
+        working-directory: openvino
+        run: |
+          cmake -B build -G Ninja -DLLVM_DIR=${LLVM_INST_PATH}/lib/cmake/llvm -DMLIR_DIR=${LLVM_INST_PATH}/lib/cmake/mlir -DENABLE_GRAPH_COMPILER=ON -DENABLE_INTEL_GPU=ON -DENABLE_TESTS=ON
+          cmake --build build --target all
+      
+      - name: Benchmark
+        working-directory: openvino
+        run: |
+          pip install openvino torch
+          for param in 'linear[512,512,512]' 'linear[1024,1024,1024]' 'linear[2048,2048,2048]', 'linear[4096,4096,4096]' 'linear[8192,8192,8192]' 'linear[4096,512,4096]'; do
+            python3 ./tools/mlir_bench/ov_model_gen.py -l=$param -t f16 -n test.xml
```

Comment:
I think the root-cause for the [error we see in CI](https://github.com/intel/graph-compiler/actions/runs/11338663184/job/31532243358#step:7:65) is this bug (https://github.com/intel/graph-compiler/issues/360)
```
loc(fused<{name = "aten::linear/Add", type = "Add"}>["aten::linear/Add"]): error: operand #0 does not dominate this use
```

The GPU pipeline is currently not working for models generated by `ov_model_gen.py`; only simple matmul modules [like this](https://github.com/dchigarev/openvino/blob/gc-gpu/src/plugins/intel_gpu/tests/functional/mlir_op/models/matmul_64_128_f16.xml) are functioning